Podfile

```diff
@@ -175,7 +175,7 @@ abstract_target 'Apps' do
   ## Gutenberg (React Native)
   ## =====================
   ##
-  gutenberg tag: 'v1.84.0'
+  gutenberg tag: 'v1.84.1'
```

Comment:
It might be redundant to merge `1.84.1` into `trunk` if we'll soon upgrade to `1.85.0` (I don't know if we will, but I assume we do based on the past schedules).

However, there was no merge conflict on the `Podfile` itself so I guess Git made the choice for us.

Reply:
> It might be redundant to merge 1.84.1 into trunk if we'll soon upgrade to 1.85.0 (I don't know if we will, but I assume we do based on the past schedules).

Yep, we'll upgrade to `1.85.0` next Thu/Fri. In any case, I'd advocate upgrading `trunk` always with the latest Gutenberg Mobile version.